### ISSUE TITLE
Question: Is Eschernet trained on Segmented Data?

### ISSUE THREAD

**Issue by hsilvaga** [kxhit/EscherNet#15]
With the gradio demo using segment anything, I'm wondering if the training data from Objaverse is also segmented?

**Comment by lorenmt** [comment]
No. Because they all have white background.

**Comment by hsilvaga** [comment]
Do you think Eschernet could be trained on unsegmented data which includes the background? Or could there be some limitations with Eschernet?

**Comment by lorenmt** [comment]
Of course. EscherNet with scene-level data is one of the most important future directions that we are currently exploring. Just simply obtaining these data is not that straightforward.
